.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
-sudo: false
+sudo: required
+services:
+  - xvfb
+env: DEPS="numpy scipy cython nose matplotlib scikit-learn"
 matrix:
     include:
     - python: "3.6"
-      env: DEPS="numpy scipy cython nose matplotlib scikit-learn"
-    - python: "3.5"                                                                                         
-      env: DEPS="numpy scipy=0.18 cython nose matplotlib scikit-learn=0.18.1"
-    - python: "2.7"                                                                                         
-      env: DEPS="numpy=1.10.4 scipy=0.17 cython nose matplotlib scikit-learn=0.17.1"
```

Comment:
I prefer keeping explicit version. The point is to check that oldest supported version work. If versions of numpy etc are not specified, only the latest ones will be tested.